cogs/sudo.py

```diff
@@ -374,6 +268,62 @@ async def gpuinfo(self, ctx: Context) -> None:
 
         await ctx.send(embed=embed)
 
+    @sudo.command()
+    @check(is_owner)
+    async def cmduser(self, ctx: Context, user: t.Union[Member, User], *, command: str) -> None:
+        """Run a command as another user."""
+        msg = copy.copy(ctx.message)
+        msg.author = user
+        msg.content = ctx.prefix + command
+
+        new_ctx = await self.bot.get_context(msg)
+        await self.bot.invoke(new_ctx)
```

Comment:
This command is unacceptable, we can't have access to run commands as other users, that way we could give ourselves any role (up to bots role) by executing commands as unknown servers owner. Even as bot developers we don't deserve the right to do anything on the server just because they have our bot there. This needs to be removed.
```suggestion
```